client/src/pages/sykmelding/ApenSykmelding/Form.tsx

```diff
@@ -0,0 +1,160 @@
+    const { formState, errors, setFormState, handleSubmit } = useForm<Form>({ validationFunctions });
+
+    return (
+        <form
+            onSubmit={(event) => {
+                event.preventDefault();
+                handleSubmit((state) => console.log(state));
+            }}
+        >
+            <RadioPanelGruppe
+                name="opplysningeneErRiktige"
+                className="margin-bottom--2"
+                legend="Er opplysningene riktige?"
+                feil={errors.get('opplysningeneErRiktige')?.feilmelding}
+                checked={
+                    formState.opplysningeneErRiktige
+                        ? 'Ja'
+                        : formState.opplysningeneErRiktige === false
+                        ? 'Nei'
+                        : undefined
+                }
+                onChange={(event, value) => {
+                    setFormState(
+                        (state): Partial<Form> => ({ ...state, opplysningeneErRiktige: value === 'Ja' ? true : false }),
+                    );
+                }}
+                radios={[
+                    { label: 'Ja', value: 'Ja', id: 'opplysningeneErRiktige' },
+                    { label: 'Nei', value: 'Nei' },
+                ]}
+            />
+
+            {formState.opplysningeneErRiktige === false && (
+                <CheckboksPanelGruppe
+                    legend={'Hvilke opplysninger stemmer ikke?'}
+                    className="margin-bottom--2"
+                    feil={errors.get('uriktigeOpplysninger')?.feilmelding}
+                    checkboxes={[
+                        {
+                            label: 'Diagnose',
+                            checked: formState.uriktigeOpplysninger?.includes('DIAGNOSE'),
+                            value: 'DIAGNOSE',
+                            id: 'uriktigeOpplysninger',
+                        },
+                        {
+                            label: 'Periode',
+                            checked: formState.uriktigeOpplysninger?.includes('PERIODE'),
+                            value: 'PERIODE',
+                        },
+                        { label: 'Annet', checked: formState.uriktigeOpplysninger?.includes('ANNET'), value: 'ANNET' },
+                    ]}
+                    onChange={(event, value) => {
+                        let updatedArray = formState.uriktigeOpplysninger ? [...formState.uriktigeOpplysninger] : [];
+                        const shouldRemoveValue = updatedArray.includes(value);
+                        if (shouldRemoveValue) {
+                            updatedArray = updatedArray.filter((opplysning) => opplysning !== value);
+                        } else {
+                            updatedArray.push(value);
+                        }
+                        setFormState((state): Partial<Form> => ({ ...state, uriktigeOpplysninger: updatedArray }));
+                    }}
```

Comment:
Jeg er ikke så glad i modifisering av array på denne måten.

I stedet for push, bruk updatedArray = [...updatedArray, value]